test/Microsoft.Azure.Devices.Edge.Test.Common/EdgeConfiguration.cs

```diff
@@ -61,39 +179,40 @@ public void AddRegistryCredentials(string address, string username, string passw
                 });
         }
 
-        // Adds proxy information to each module in Edge Agent's desired properties. Call this
-        // method after you've added all the modules that need proxy information.
-        public void AddProxy(Uri proxy)
+        public Module AddEdgeAgent(string image)
```

Comment:
This public API is returning a nested type.. seems a bit confusing. Consider un-nesting the class

Reply:
Sure I can do that.

Reply:
This file was deleted in the latest round of changes.